### DISCUSSION TITLE
Modal Dialog component with Bootstrap in Blazor

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#123] [General]
# Modal Dialog component with Bootstrap in Blazor

This short blog post will show you how to utilize Bootstrap to create a small and reuseable ModalDialogComponent.

https://steven-giesel.com/blogPost/5fc5b957-d62e-40e6-b0c4-f2a0df5c8aa1

**Comment by woocache** [comment]
Great example. Do you know how add the fading animation same as in bootstrap (modal fade).

**Reply by linkdotnet** [reply]
To my understanding the `modal fade` classes in combination should work just fine: https://getbootstrap.com/docs/5.3/components/modal/#varying-modal-content

I am no expert on this, but it is "just" setting the correct `data-bs` toggles/targets.
